### ISSUE TITLE
Add Plausible for tracking metrics

### ISSUE THREAD

**Issue by pavithraes** [conda-incubator/conda-store#408]
Plausible (https://plausible.io/) is similar to Google Analytics, and we're using it on other Quansight project websites as well.

It'll help us track who visits our docs pages, and how often.

**Comment by trallard** [comment]
We can do this after #397

**Comment by trallard** [comment]
On hold until we merge #535

**Comment by kcpevey** [comment]
#534 has been merged so this is now unblocked.

**Comment by trallard** [comment]
@pavithraes shall we add Plausible to the existing conda?store docs? Or shall we wait until we have the new and updated docs?

**Comment by pavithraes** [comment]
I think we can wait for the new docs, but we can add Plausible once we have the initial set of pages ready
